sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/implementation/KeyVaultCredentialPolicy.java

```diff
@@ -208,6 +208,7 @@ public void authorizeRequestSync(HttpPipelineCallContext context) {
                 .setTenantId(this.challenge.getTenantId());
 
             setAuthorizationHeaderSync(context, tokenRequestContext);
+            return;
```

Comment:
Note: This is required, it was missing before when this policy got updated via #29730